tasks/release.rb

```diff
@@ -242,8 +241,6 @@ def rc?
       require "date"
       future_date = Date.today + 5
       future_date += 1 while future_date.saturday? || future_date.sunday?
-
-      github_user = `git config github.user`.chomp
```

Comment:
`github_user` is used in https://github.com/rails/rails/blob/master/tasks/release_announcement_draft.erb

Reply:
@kamipo thanks!